templates/taxi/manufacturer_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="🔎" class="btn btn-secondary">
+  </form>
```

Comment:
Create a separate template for search form and include it where you need to avoid code duplication